archived/challengesObject.js

```
@@ -0,0 +1,17 @@
+  }
+
+  static greet(name) {
+    console.log(`Hi, ${name}!`);
+  }
+
+  static calculateFreelancerEarnings(hourlyRate, hoursWorked) {
+    const rate = parseFloat(hourlyRate);
+    const hours = parseFloat(hoursWorked);
+    return rate * hours;
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)